src/extensions/hive/components/database/SelectDatabase.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@mui/styles';
+import { Tabs, Tab, Typography } from '@mui/material';
+import { UploadDatabase } from './UploadDatabase';
+import { ConfigureSelfManagedDatabase } from './ConfigureSelfManagedDatabase';
+import { TabPanel } from '../tabs/TabPanel';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+  },
+  tabs: {},
+}));
+
+export const SelectDatabase = (props) => {
+  const { existingDbName, connection, setConnection, theme } = props;
+
+  const [tabIndex, setTabIndex] = useState(0);
+
+  const handleTabChange = (event, newValue) => {
+    setTabIndex(newValue);
+  };
+
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <ConfigureSelfManagedDatabase connection={connection} setConnection={setConnection} />
+      {/*
+      <Tabs
```

Comment:
Remove commented code?

Reply:
will do

Reply:
Removed comments, and then removed Upload, Dropzone, and InlineBox and associated css files